packages/docusaurus-theme-common/src/utils/announcementBarUtils.tsx

```diff
@@ -96,7 +95,9 @@ const useAnnouncementBarContextValue = (): AnnouncementBarAPI => {
   );
 };
 
-const AnnouncementBarContext = createContext<AnnouncementBarAPI | null>(null);
+const AnnouncementBarContext = React.createContext<AnnouncementBarAPI | null>(
```

Comment:
not against this change, just wondering why it's necessary?

Reply:
Oh, purely stylistic. We have some `React.createContext` and some `createContext`. I personally prefer to have the `React.` prefix except for hooks so we understand what we are talking about